### PR TITLE
Add preset for aws scale account

### PR DESCRIPTION
**Pull request by prateekgogia**

### CODE REVIEW

File: config/jobs/kubernetes/sig-cloud-provider/aws/sig-aws-credentials.yaml [hunk 1]
```diff
@@ -28,3 +28,18 @@ presets:
...
+  - name: AWS_SHARED_CREDENTIALS_FILE
+    value: /etc/aws-cred/credentials
+  volumeMounts:
+  - name: aws-cred
+    mountPath: /etc/aws-cred
+    readOnly: true
+  volumes:
+  - name: aws-cred
+    secret:
+      secretName: aws-credentials-boskos-scale-001-kops
```
Comment (xmudrii) [inline config/jobs/kubernetes/sig-cloud-provider/aws/sig-aws-credentials.yaml:45]: Please add a new line

File: config/jobs/kubernetes/sig-cloud-provider/aws/sig-aws-credentials.yaml [hunk 2]
```diff
@@ -28,3 +28,18 @@ presets:
   - name: aws-cred
     secret:
       secretName: aws-credentials-607362164682
+
+# Credentials for using AWS account `k8s-infra-e2e-scale-boskos-001`. Used for scale tests in AWS.
+- labels:
+    preset-aws-credential-aws-scale-testing: "true"
+  env:
+  - name: AWS_SHARED_CREDENTIALS_FILE
+    value: /etc/aws-cred/credentials
```
Comment (xmudrii) [inline config/jobs/kubernetes/sig-cloud-provider/aws/sig-aws-credentials.yaml:37]: Please change this to:

```suggestion
    value: /etc/aws-cred/sharedCredentialsFile
```

As per https://github.com/kubernetes/k8s.io/pull/5676

**Comment by ameukam** [comment]
/hold

This new preset should only be introduced to scalability tests. Why are we mutating all the kops jobs ?

**Comment by prateekgogia** [comment]
> This new preset should only be introduced to scalability tests. Why are we mutating all the kops jobs ?

build-jobs.py script auto updates the jobs with latest images, I have reverted that change for now to keep the changes limited for preset configurations.

**Pull request merged**